com.unity.render-pipelines.high-definition/Documentation~/Ray-Traced-Global-Illumination.md

```diff
@@ -14,9 +14,12 @@ For information about ray tracing in HDRP, and how to set up your HDRP Project t
 
 ## Using Ray-Traced Global Illumination
 
-1. Because this feature replaces the [Screen Space Global Illumination](Override-Screen-Space-GI.md) Volume override, the initial setup is very similar. To setup ray traced global illumination, first follow the [Enabling Screen Space Global Illumination](Override-Screen-Space-GI.md#enabling-screen-space-global-illumination) and [Using Screen Space Global Illumination](Override-Screen-Space-GI.md#using-screen-space-global-illumination) steps. After you setup the Screen Space Global Illumination override, to make it use ray tracing:
-   1. In the Frame Settings for your Cameras, enable **Ray Tracing**.
-   2. Select the [Screen Space Global Illumination](Override-Screen-Space-GI.md) override and, in the Inspector, enable **Ray Tracing**. If you do not see a **Ray Tracing** option, make sure your HDRP Project supports ray tracing. For information on setting up ray tracing in HDRP, see [Getting started with ray tracing](Ray-Tracing-Getting-Started.md).
+This feature replaces the [Screen Space Global Illumination](Override-Screen-Space-GI.md) Volume override, so the initial setup is similar. To setup ray traced global illumination:
+
+1. Follow the [Enabling Screen Space Global Illumination](Override-Screen-Space-GI.md#enabling-screen-space-global-illumination) and [Using Screen Space Global Illumination](Override-Screen-Space-GI.md#using-screen-space-global-illumination) steps to set up the Screen Space Global Illumination override. 
+2. In the Frame Settings for your Cameras, enable **Ray Tracing**.
```

Comment:
Consider: Does the user have enough information here to change this setting?

Try it yourself in Unity and see.